cfpb/consumerfinance.gov
# Update default video thumbnail to better fit dimensions

## Changes

- Adjust default no-js video thumbnail image dimensions to show the CFPB logo at all screensizes.


## How to test this PR

1. Pull branch and run gulp clean && gulp build
2. Visit a video page, such as http://localhost:8000/consumer-tools/everyone-has-a-story/leah-repaying-student-loans/, http://localhost:8000/consumer-tools/reverse-mortgages/, http://localhost:8000/coronavirus/avoiding-scams/
3. Open the dev console settings and disable JavaScript.
4. Reload the page to show the default video image.
5. Resize the window and note that the image is cropped at certain sizes.


## Screenshots

Before:
<img width="1273" alt="Screen Shot 2021-02-23 at 2 25 38 PM" src="https://user-images.githubusercontent.com/704760/108896512-06f9eb00-75e3-11eb-947f-92f6761926d7.png">

After:
<img width="1051" alt="Screen Shot 2021-02-23 at 1 50 11 PM" src="https://user-images.githubusercontent.com/704760/108896470-f6497500-75e2-11eb-8bc2-4798c00cc511.png">

Before:
![video-default-old](https://user-images.githubusercontent.com/704760/108896559-15480700-75e3-11eb-9ce5-cca59f55f424.gif)

After:
![video-default](https://user-images.githubusercontent.com/704760/108896573-1bd67e80-75e3-11eb-9377-c46b70f72f68.gif)
